<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SyncSummary</class>
 <widget class="QDialog" name="SyncSummary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(20,20,20);
color: white;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="new_files_text">
     <property name="text">
      <string>New files</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QScrollArea" name="new_files_scrollarea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="new_files_widget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>380</width>
        <height>90</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="updated_files_text">
     <property name="text">
      <string>Updated files</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QScrollArea" name="updated_files_scrollarea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="updated_files_widget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>380</width>
        <height>90</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3"/>
     </widget>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="upgrade_button">
       <property name="minimumSize">
        <size>
         <width>96</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>96</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Upgrade</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
